Clean up whitespace

AArnott opened the pull request:
We had mixed tabs and spaces, and lots of trailing whitespace. When whitespace is visible (which is my default), it shows up and is quite distracting.

AArnott committed:
Clean up whitespace

We had mixed tabs and spaces, and lots of trailing whitespace. When whitespace is visible (which is my default), it shows up and is quite distracting.

rmarinho commented:
Hum, but we use tabs and this replaces for spaces no?

mattleibow commented:
Not for this XAML I think. Should we use tabs, maybe. But, I do know people like to indent weird in XAML so spaces are usually better here.

```xml
<TheThing First=""
          Second="" />
```

AArnott commented:
@rmarinho The precedent in the repo is tabs for .cs files and spaces for .xaml. I didn't change any indentation anyway. I just deleted trailing whitespace. It just so happened that this xaml file contained a mix of tabs and spaces in that trailing whitespace. I deleted all of them equally.
